example/package.json

```diff
@@ -7,8 +7,8 @@
     "sync": "rm -rf ./node_modules/react-native-gifted-chat; sane '/usr/bin/rsync -v -a --exclude .git --exclude example --exclude __tests__ --exclude node_modules ../ ./node_modules/react-native-gifted-chat/' .. --glob='{**/*.json,**/*.js}'"
   },
   "dependencies": {
-    "react": "15.2.1",
-    "react-native": "^0.30.0",
+    "react": "~15.4.0-rc.4",
```

Comment:
you can see the peerDependency here [https://github.com/facebook/react-native/blob/master/package.json](https://github.com/facebook/react-native/blob/master/package.json)